sw/device/sca/lib/sca.c

```diff
@@ -293,3 +293,17 @@ void sca_call_and_sleep(sca_callee callee, uint32_t sleep_cycles) {
   OT_DISCARD(dif_clkmgr_gateable_clock_set_enabled(
       &clkmgr, CLKMGR_CLK_ENABLES_CLK_IO_DIV4_PERI_EN_BIT, kDifToggleEnabled));
 }
+
+/**
+ * PRNG for masking key.
+ */
+static uint32_t lfsr_state = 0xdeadbeef;
+
+uint32_t next_lfsr(void) {
+  const uint32_t lfsr_out = lfsr_state;
+  bool lfsr_bit = lfsr_state & 0x00000001;
+  lfsr_state = lfsr_state >> 1;
+  if (lfsr_bit)
```

Comment:
Could you add braces to this `if` for complience with the coding [style](https://docs.opentitan.org/doc/rm/c_cpp_coding_style/)?